docs/k8s-quickstart.asciidoc

```diff
@@ -292,7 +292,7 @@ spec:
         resources:
           requests:
             storage: 10Gi
-        storageClassName: gcePersistentDisk # can be any available storage class
+        storageClassName: standard # can be any available storage class
```

Comment:
I wonder if we should keep the whole line commented because using the `standard` storageClassName is the default behaviour?

Reply:
I think I agree with @thbkrkr